src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -10,30 +10,49 @@ public class HelpCommand extends Command {
     public static final String COMMAND_WORD = "help";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-        + "Example: " + COMMAND_WORD;
+            + "Example: " + COMMAND_WORD;
 
     // public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
     // Give a more informative help message that explains how to use basic commands
     public static final String HELP_MESSAGE = "Ensure you have registered an account / have logged"
-        + " in before using the following basic commands.\n\nRefer to the URL in the pop-up window for"
-        + " our full user guide.\n"
-        + "\t1. register: Registers a new account.\n"
-        + "\t2. login: Logs in to an existing account.\n"
-        + "\t3. logout: Logs out of the current account.\n"
-        + "\t4. help: Shows program usage instructions.\n"
-        + "\t5. exit: Exits the program.\n"
-        + "\t6. clear: Clears the screen.\n\n"
-
-        + "The following commands can be used after logging in.\n"
-        + "\tFor customers:\n"
-        + "\t1. customer list: Lists all customers.\n"
-        + "\t2. customer add: Adds a customer.\n"
-        + "\t3. customer delete: Deletes a customer.\n"
-        + "\t4. customer edit: Edits a customer.\n\n"
-
-        + "\tFor deliveries:\n"
-        + "\t1. delivery status: Lists all deliveries.\n\n\n"
-        + "More commands will be added in the future.";
+            + " in before using the following basic commands, except for the ones indicated with a `.\n\n"
+            + "USER\n"
+            + "These are commands for managing your user account in HomeBoss.\n"
+            + "* `register - Registers a new user account to use HomeBoss.\n"
+            + "* `login - Logs in to your user account.\n"
+            + "* update - Updates your user account details.\n"
+            + "* `recover account - Recovers your user account.\n"
+            + "* `logout - Logs out of your user account.\n"
+            + "* `delete account - Deletes your user account.\n\n"
```

Comment:
Why are there some that starts with ` and doesn't end with ` ? Copy paste error?

Reply:
The comment at the top of the help message explains it; These were meant to differentiate between commands that can be used before logged in / account is registered and the rest